### ISSUE TITLE
LSTM

### ISSUE THREAD

**Issue by lngao** [meijieru/crnn.pytorch#73]
No softmax after the lstm but just linear layer, but the lua version CRNN use LogSoftMax

**Comment by meijieru** [comment]
Yeah, the CTC I used requires logits rather than logsoftmaxed one. Also, CTC considers the probability of total sequence instead of prob of single characters, so we do not need them.
